SalesforceMobileSDK-iOS.podspec

```diff
@@ -52,7 +52,8 @@ Pod::Spec.new do |s|
       smartstore.dependency 'SalesforceMobileSDK-iOS/SalesforceSDKCore'
       smartstore.dependency 'SQLCipher', '~> 3.1'
       smartstore.dependency 'SQLCipher/fts', '~> 3.1'
-      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h', 'external/fmdb/src/fmdb/*.{h,m}'
+      smartstore.dependency 'FMDB', '~> 2.5'
+      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h'
```

Comment:
We'll need to change the way Cordova consumes the binaries and the update
script too then.

On Tuesday, December 22, 2015, Kevin Hawkins <notifications@github.com>
wrote:

> In SalesforceMobileSDK-iOS.podspec
> <https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1261#discussion_r48317318>
> :
>
> > @@ -52,7 +52,8 @@ Pod::Spec.new do |s|
> >        smartstore.dependency 'SalesforceMobileSDK-iOS/SalesforceSDKCore'
> >        smartstore.dependency 'SQLCipher', '~> 3.1'
> >        smartstore.dependency 'SQLCipher/fts', '~> 3.1'
> > -      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h', 'external/fmdb/src/fmdb/*.{h,m}'
> > +      smartstore.dependency 'FMDB', '~> 2.5'
> > +      smartstore.source_files = 'libs/Smartstore/Smartstore/Classes/**/*.{h,m}', 'libs/Smartstore/Smartstore/Smartstore.h'
>
> For what it's worth, I verified that encryption is in fact *not* working
> for an app that uses the SmartSync pod spec in its current state. I'm
> going to look at re-integrating FMDB into SmartStore, if there aren't
> objections.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1261/files#r48317318>
> .
>


-- 
Sent from my iPhone